tests/easyInterface/Utils/test_Helpers.py

```diff
@@ -55,3 +55,24 @@ def fun():
     assert fun.calls == 1
     f = fun()
     assert fun.calls == 2
+
+
+def test_createReleaseNotes():
+    def writeRL(*args):
+        createReleaseNotes('easyInterface/Release.json', *args)
+        if len(args) == 0:
+            file = 'CHANGELOG.txt'
+        else:
+            file = args[0]
+        try:
+            with open(file, 'r') as file_reader:
+                file_contents = file_reader.read()
+            assert '# easyInterface' in file_contents
+            assert '* ' in file_contents
+            os.remove(file)
+        except FileNotFoundError:
+            print('File not found :-(')
```

Comment:
Shouldn't you assert here? If file is not present, this means the test failed, I believe.

Reply:
Removed the try catch and implemented:
`assert os.path.isfile(file)`
before file opening.